imports/api/server/methods.tests.js

```diff
@@ -0,0 +1,160 @@
+import { Meteor } from 'meteor/meteor';
+import { assert, expect } from 'meteor/practicalmeteor:chai';
+import { resetDatabase } from 'meteor/xolvio:cleaner';
+import { Factory } from 'meteor/dburles:factory';
+import { student, company } from '/imports/api/seed/users.js';
+import { job } from '/imports/api/seed/jobs.js';
+import { sinon } from 'meteor/practicalmeteor:sinon';
+import '/imports/api/server/methods.js';
+
+describe('Application methods', () => {
+  beforeEach(() => {
+    resetDatabase();
+  });
+
+  describe('Inserting Company', () => {
+    it('successfully inserts a company into the users collection', () => {
+      let id = Meteor.call('saveCompany', company);
```

Comment:
The object company I'm using as parameter is the one I told you that we are using to seed the database, we are importing in at the top of the file with:

`import { student, company } from '/imports/api/seed/users.js';`